test/cpp/qps/driver.cc

```diff
@@ -32,6 +32,7 @@
 #include <grpcpp/client_context.h>
 #include <grpcpp/create_channel.h>
 
+#include "google/protobuf/timestamp.pb.h"
```

Comment:
Try using "proto.h" instead of "pb.h" (it is a smaller file).

Reply:
I have tried to compile timestamp.proto, seems like outside google it only generate the .pb.h file, could not find a .proto.h file.